returnn/torch/engine.py

```diff
@@ -669,6 +671,28 @@ def forward_with_callback(self, *, dataset: Dataset, callback: ForwardCallbackIf
 
             callback.finish()
 
+    @staticmethod
+    def delete_model(filename):
+        """
+        :param str filename:
+        :return: accumulated file-size in bytes of deleted files
+        :rtype: int
+        """
+        # This assumes PyTorch models here.
+        # They consist of a file with the extension ".pt".
+        from glob import glob
+
+        count_bytes = 0
+        assert os.path.exists(filename + ".pt")
+        for fn in glob(filename + "*"):
```

Comment:
Why this? This looks potentially dangerous to me. I think we should iterate over exactly the potential available files which we expect.

Also, from the code below, it looks like you simply want anyway this here?
```suggestion
        for fn in glob(filename + "*.pt"):
```

Reply:
This was from the tf variant of the function. But I guess there we need to check for multiple different file endings. So if I am not mistaken 
```
            if fn_ext not in [".pt"]:
                continue
```

can just be dropped right?

Reply:
Yes that can be dropped. But actually, as I said, I would drop the whole `glob` here. Or why is it necessary?